lib/quicly.c

```diff
@@ -5481,8 +5481,10 @@ int quicly_accept(quicly_conn_t **conn, quicly_context_t *ctx, struct sockaddr *
         goto Exit;
     next_expected_pn = 0; /* is this correct? do we need to take care of underflow? */
     if ((ret = decrypt_packet(ingress_cipher.header_protection, aead_decrypt_fixed_key, ingress_cipher.aead, &next_expected_pn,
-                              packet, &pn, &payload)) != 0)
+                              packet, &pn, &payload)) != 0) {
+        ret = QUICLY_ERROR_DECRYPTION_FAILED;
```

Comment:
Would it be worth triggering the `packet_decryption_failed` probe here?

```
probe packet_decryption_failed(struct st_quicly_conn_t *conn, int64_t at, uint64_t pn);
```

Reply:
I do not think we can, because the connection state has not been established yet. To paraphrase, this path is more like receiving a packet that is not bound to any connection.